doc/news/changes.h

```diff
@@ -367,6 +367,12 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> Improved: Including dofs for individual components on boundary in 
+   make_vertex_patches
```

Comment:
Can you qualify this function with its namespace so that DoFTools properly links to it? We typically also write function names as `NamespaceName::function_name()`, i.e., with parentheses at the end.